sdcm/utils/loader_utils.py

```diff
@@ -117,6 +117,55 @@ def _run_all_stress_cmds(self, stress_queue, params):
 
         return stress_queue
 
+    @staticmethod
+    def parse_cs_user_profiles_param(cs_user_profiles):
+        """
+        The function parses user_profile parameter that defined in test yaml as:
+        ```
+            cs_user_profiles:
+                - scylla-qa-internal/custom_d1/rolling_upgrade_dataset.yaml,45m
+                - scylla-qa-internal/custom_d1/rolling_upgrade_dataset2.yaml,45m
+        ```
+        where first is user profile and second - how long the profile will be run (optional)
+        The goal is to create two lists:
+        - list of cs user profiles
+        - list of duration for every cs user profile (if defined)
+        The expected output of the function:
+            user_profiles = ['scylla-qa-internal/custom_d1/rolling_upgrade_dataset.yaml',
+                             'scylla-qa-internal/custom_d1/rolling_upgrade_dataset2.yaml']
+            duration_per_cs_profile = ['45m', '45m']
+        """
+        user_profiles_def = list(zip(*[profile.split(",") for profile in cs_user_profiles]))
+        user_profiles = list(user_profiles_def[0])
+        duration_per_cs_profile = None if len(user_profiles_def) == 1 else list(user_profiles_def[1])
+        return user_profiles, duration_per_cs_profile
+
+    def run_cs_user_profiles(self, cs_profiles: str | int | list, duration_per_cs_profile: str | list = None, stress_queue: list = None):  # pylint: disable=too-many-locals
```

Comment:
'format' is used in 2 ways - with or without duration. Add docstring what is expected from c-s command in profile file with some example, so when using this method is clear without searching for examples.

Reply:
added docstring